### DISCUSSION TITLE
Deluge plugin: move_completed_path stopped working.

### DISCUSSION THREAD

**Discussion by brunoga** [Flexget/Flexget#3285] [Q&A]
It seems that flexget is setting the move completed flag in Deluge (it is checked for the torrents being downloaded) but the actual path is not set (it ends up being empty so files are not moved after download). Any ideas what might be happening? It used to work but I guess I upgraded several things (most notably going to Ubuntu Impish). One thing I did *NOT* change was my Flexget configuration. It is the same as it was when it was actually working.

Thanks in advance.

Flexget version is 3.2.4 and Deluge is 2.0.3.

**Comment by girtsl** [comment]
I believe it has something to do with your changes. I'm currently on flexget 3.2.8 and deluge 2.0.5 and the `move_completed_path` has never once stopped working for me. 
Sorry I can't offer any advice on what exactly might be wrong.
